src/action/timeline.rs

```diff
@@ -166,12 +174,66 @@ impl super::Response for ChunkResponse {
 
     const RDF_NAME: Option<&'static str> = Some("DataBlob");
 
-    type Proto = rrg_proto::DataBlob;
+    type Proto = DataBlob;
 
-    fn into_proto(self) -> rrg_proto::DataBlob {
-        rrg_proto::DataBlob {
+    fn into_proto(self) -> DataBlob {
+        DataBlob {
             data: Some(self.data),
             ..Default::default()
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
This test suite is not very exhaustive. The timeline action is actually one of the few actions there are very easy to test, including all corner cases (e.g. empty directories, deeply nested file hierarchies, files of different types, symlinks with cycles, hardlinks etc.).

In general, there should be _a lot_ of test cases for this action, you can get really wild here.